include/seqan/misc/union_find.h

```diff
@@ -298,7 +298,8 @@ resizeVertexMap(UnionFind<TValue> & unionFind,
                 Graph<TSpec> const & g)
 {
     clear(unionFind);
-    return resize(unionFind, numVertices(g));
+    resizeVertexMap(unionFind._values, g, -1);
+    return length(unionFind);		// Note: Other resizeVertexMap() overloads return void...
```

Comment:
True. I might change this function to `void` in the future for consistency and the returned value is not used throughout the SeqAn library anyway. Thank you for pointing that out.

Reply:
You're welcome :)